challenge2Fizzbusstest.js

```diff
@@ -0,0 +1,14 @@
+function FizzBuzz(a, b) {

```

Comment:
The logic looks correct but I will need to look at the question, please add it in the description of your pull request